### ISSUE TITLE
Scanner cannot read input

### ISSUE THREAD

**Issue by RobvH** [albertlatacz/java-repl#107]
Basic "hello {you}" flow doesn't work. Calling scan.next() just blocks but without accepting input or recognizing EOL.

**Comment by albertlatacz** [comment]
Is that on System.in?

**Comment by RobvH** [comment]
Yes.

**Comment by albertlatacz** [comment]
Yeah that might be tricky. Basically javarepl is a web service so more of request-response rather than stream
